whisper/src/rpc/mod.rs

```diff
@@ -339,7 +329,7 @@ impl<P: PoolHandle + 'static, M: Send + Sync + 'static> Whisper for WhisperClien
 			payload: encrypted,
 			topics: req.topics.into_iter().map(|x| abridge_topic(&x.into_inner())).collect(),
 			work: req.priority,
-		});
+		}).map_err(|_| whisper_error("Empty topics"))?;
```

Comment:
Is it the only possible reason for `Message::create` to fail?

Reply:
Yep, why?

Reply:
Just asking, because it's far from obvious in the context